doc/src/explore/tutorials.md

```diff
@@ -24,10 +24,11 @@ remainder of this tutorial.
 We will follow the same convention as the one described in the [Sui
 setup instructions](../build/wallet.md#setup) and assume that Sui
 configuration files generated durind Sui genesis state creation are
-stored in the `"$SUI_ROOT"/sui_instance` directory, and **for the
-remainder of this tutorial we will assume that you are executing the
-`wallet` command in this directory**.
+stored in the `"$SUI_ROOT"/sui_instance` directory.
 
+*IMPORTANT*: For the remainder of this tutorial, we will assume that you are
+executing the `wallet` command in the `"$SUI_ROOT"/sui_instance` directory.
```

Comment:
```suggestion
executing the `wallet` command in the `"$SUI_ROOT"/sui_instance` directory as well.
```

Without it the end of the last paragraph and the current paragraph read a bit redundant to me.